package.json

```diff
@@ -47,6 +48,7 @@
     "pg-format": "^1.0.4",
     "randomstring": "^1.1.5",
     "regenerator-runtime": "^0.13.3",
+    "safe-json-stringify": "^1.2.0",
```

Comment:
could you use [json-stringify-safe](https://www.npmjs.com/package/json-stringify-safe) instead? it seems to be much more popular than safe-json-stringify in terms of downloads (10x more)

Reply:
either one works, but we probably might want to do the same in general-runtime:
https://github.com/voiceflow/general-runtime/blob/c775602830e26fa4010f091ea021b700a3f3c67d/package.json#L54